src/core/services/interimElement/interimElement.js

```diff
@@ -291,6 +291,12 @@ function InterimElementProvider() {
         // This hide()s only the current interim element before showing the next, new one
         // NOTE: this is not reversible (e.g. interim elements are not stackable)
 
+        if (!hideExisting) {
+          return $q(function(resolve, reject) {
+            reject("Too many interim elements in queue!");
```

Comment:
Not sure about the rejection, but that's the best way to prevent the interim element to stuck on showing. Any suggestions?